scraper/src/strategies/default_strategy.py

```diff
@@ -190,24 +195,36 @@ def _generate_empty_hierarchy():
     @staticmethod
     def _get_level_of_node(node, nodes_per_level, levels):
         for level in levels:
-            if node in nodes_per_level[level]:
-                return level
+            if level in nodes_per_level: # if it's global we won't have it
+                if node in nodes_per_level[level]:
+                    return level
 
         return None
 
     @staticmethod
-    def _handle_default_values(hierarchy, content, selectors):
+    def _handle_default_values(hierarchy, content, selectors, levels):
         # Handle default values
         for level in selectors:
-            if level != 'content':
+            if level in levels:
                 if hierarchy[level] is None and selectors[level]['default_value'] is not None:
                     hierarchy[level] = selectors[level]['default_value']
-            else:
+            elif level == 'content':
                 if content is None and selectors[level]['default_value'] is not None:
                     content = selectors['content']['default_value']
 
         return hierarchy, content
 
+    def _get_nodes_per_global_level(self, selectors, levels):
+        for level in selectors.keys():
+            level_selector = selectors[level]
+            if level not in levels or level_selector['global']:
```

Comment:
better to check a boolean first? iterating cost more (short-circuit)

Reply:
👍